chain-impl-mockchain/src/vote/manager.rs

```diff
@@ -1377,18 +1389,15 @@ mod tests {
             .vote(identifier.clone(), second_vote_cast.payload.clone())
             .unwrap();
 
-        let mut stake_controlled = StakeControl::new();
-        stake_controlled =
-            stake_controlled.add_to(identifier.to_single_account().unwrap(), Stake(51));
-        stake_controlled = stake_controlled.add_unassigned(Stake(49));
+        let (ledger, token) = ledger_with_tokens(identifier.clone());
```

Comment:
nit: not immediately clear why this works

Also now 100% of tokens are voting in favor of the proposal, not sure if it matters here though

Reply:
> nit: not immediately clear why this works

what part exactly? maybe I should add an extra account that doesn't cast any vote to compensate for that `unassigned` stake? Because there is no more `unassigned` stake anymore.

Reply:
> what part exactly?

ops, forget this, it's a leftover